app/population/constants.js

```diff
@@ -0,0 +1,25 @@
+const { dateRegex } = require('../../common/helpers/url')
+
+const MOUNTPATH = '/population'
+
+const BASE_PATH = `/:period(week|day)/:date(${dateRegex})`
+
+const DEFAULTS = {
+  QUERY: {
+    population: { status: '', sortBy: 'date', sortDirection: 'asc' },
+  },
+  TIME_PERIOD: {
+    population: 'week',
+  },
+}
+
+const FILTERS = {
+  filterTypething: 'prison',
```

Comment:
Wonder if there might be better name for this... maybe `locationType`?

Reply:
Based on the comments below about the intended use of filters, this code block has been removed.

Also, that was an appalling name for a variable! :joy: